superset/views/core.py

```diff
@@ -1747,6 +1747,8 @@ def favstar(  # pylint: disable=no-self-use
         self, class_name: str, obj_id: int, action: str
     ) -> FlaskResponse:
         """Toggle favorite stars on Slices and Dashboard"""
+        if not g.user.get_id():
+            return json_error_response(f"ERROR: Favstar toggling denied", status=403)
```

Comment:
the linter doesn't like that this is an f-string, it should just be a regular string

Reply:
thank you, resolved this in 07e6e6c + includes your suggestion for favorites filter cast.